src/js/components/ServerErrorModal.js

```diff
@@ -56,7 +56,7 @@ module.exports = class ServerErrorModal extends mixin(StoreMixin) {
     });
   }
 
-  handleServerError(errorMessage) {
+  handleServerError({message:errorMessage}) {
```

Comment:
As @wavesoft mentioned we are using the parseResponseBody because the message does not give us enough information about the error. The message for some errors in marathon are for example `Invalid Format` if we surface that error the user does not know which field is triggering this error. In the details is a array containing a detailed information. The following JSON should show this dilemma:
```JSON
{
   "message":"Object is not valid",
   "details":[
      {
         "path":"/",
         "errors":[
            "AppDefinition must either contain one of 'cmd' or 'args', and/or a non-Mesos 'container'."
         ]
      }
   ]
}
```

But I agree that we need a single way to parse errors, but it should be more verbose.